src/Entity/Timeline/Theme.php

```diff
@@ -85,59 +53,33 @@ class Theme implements EntityMediaInterface
      */
     private $measures;
 
+    /**
+     * @Assert\Valid
+     */
+    protected $translations;
```

Comment:
Why using `protected` here?

Reply:
i was following the documentation blindly.
i can make it private